src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -89,6 +89,8 @@ private void OnInitialized(object sender, EventArgs e)
 
         private void OnLoaded(object sender, System.Windows.RoutedEventArgs _)
         {
+            WindowsInteropHelper.DisableControlBox(this);

```

Comment:
Is this basically disabling the menu that appears with Alt+Space, so that Alt+Space doesn't trigger that menu for Launcher?

Reply:
yes